src/components/docs-anchor/index.tsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable jsx-a11y/anchor-has-content -- We're just passing through props here, so the links should be well formatted */
+import { ProductSlug } from 'types/products'
+import Link from 'next/link'
+
+export default function makeDocsAnchor(
+  product: ProductSlug,
+  basePaths: string[]
+) {
+  const DocsAnchor: React.FC<JSX.IntrinsicElements['a']> = ({
+    href,
+    ...rest
+  }) => {
+    if (!href) return <a {...rest} />
```

Comment:
Super nitty but instead of disabling `anchor-has-content`, I think you could do:

```suggestion
    href,
    children,
    ...rest
  }) => {
    if (!href) return <a {...rest}>{children}</a>
```

☝️  Not actually sure this is preferable, just wanted to put the idea out there.

Reply:
It worked! Great call.